vowpalwabbit/cb_algs.cc

```diff
@@ -83,7 +88,12 @@ void learn_eval(cb& data, single_learner&, example& ec)
   CB_EVAL::label ld = ec.l.cb_eval;
 
   cb_to_cs& c = data.cbcs;
-  c.known_cost = get_observed_cost(ld.event);
+  auto optional_cost = get_observed_cost_cb(ld.event);
```

Comment:
same here? just assign `optional_cost.second` to known cost? Maybe instead of `optional_cost` the name `observed_or_default_cost` would also work

Reply:
Same explanation here too